sdk/js/src/Client.ts

```diff
@@ -200,10 +200,11 @@ export class DVCClient implements Client {
 
             let updatedUser: DVCPopulatedUser
 
+            const storedAnonymousId = this.store.loadAnonUserId()
```

Comment:
we could move this to the else block, if this is only used there, that way we only read from the cache when we need to